Bug-fix FastJetFinder.cc in fComputeTrimming

jared-burleson opened the pull request:
Fixed a bug with the Trimming Method where individual subjets would be constructed with only 1 constituent, now allows for subjets with multiple constituents.

The edit was only in /modules/FastJetFinder.cc on line 492.

selvaggi commented:
thanks for fix @jared-burleson. Could you remove all the comments from the code and simply add them in the PR comments?

jared-burleson commented:
Previously FastJetFinder.cc generated a trimmed_jet where each piece (subjet) has a single constituent only. This isn't necessarily correct, a subjet could have multiple constituents, according to defining trimming paper. This fix retains the trimmed_jet P4 but also allows subjets to be formed with multiple constituents based on the R-parameter requirements, and is consistent with the ATLAS/ATHENA code from their github respository.

jared-burleson commented:
> thanks for fix @jared-burleson. Could you remove all the comments from the code and simply add them in the PR comments?

Removed the comments in the code and added the above comment to explain the change in the code. Let me know if there is anything else needed.
